src/audio/base_fw.c

```diff
@@ -9,13 +9,25 @@
 #include <ipc4/logging.h>
 #include <sof_versions.h>
 
+#if CONFIG_ACE_V1X_ART_COUNTER || CONFIG_ACE_V1X_RTC_COUNTER
+#include <zephyr/device.h>
+#include <zephyr/drivers/counter.h>
+#endif
+
 LOG_MODULE_REGISTER(basefw, CONFIG_SOF_LOG_LEVEL);
 
 /* 0e398c32-5ade-ba4b-93b1-c50432280ee4 */
 DECLARE_SOF_RT_UUID("basefw", basefw_comp_uuid, 0xe398c32, 0x5ade, 0xba4b,
 		    0x93, 0xb1, 0xc5, 0x04, 0x32, 0x28, 0x0e, 0xe4);
 DECLARE_TR_CTX(basefw_comp_tr, SOF_UUID(basefw_comp_uuid), LOG_LEVEL_INFO);
 
+static struct ipc4_system_time_info global_system_time_info = {
+		.host_time.val_l = 0,
+		.host_time.val_u = 0,
+		.host_time.val_l = 0,
+		.host_time.val_u = 0
+};
```

Comment:
I believe this is unnecessary. BSS section will be zeroed.

Reply:
@kv2019i ok, I wasn't sure before. Removed.